backend/rebutify/settings.py

```diff
@@ -81,7 +81,8 @@
 ]
 
 # Normalizing URLs for front-end (https://docs.djangoproject.com/en/4.0/ref/middleware/#django.middleware.common.CommonMiddleware)
-APPEND_SLASH = False
+APPEND_SLASH = True
+PREPEND_WWW = True
```

Comment:
on second thought PREPEND_WWW should be left out if we want to eventually make the backend api.rebutify.org
so it will get even shorter
1 line PR?

Reply:
Okay. We will set PREPEND_WWW to False for now.